src/com/serotonin/mango/util/LoggingScriptUtils.java

```diff
@@ -0,0 +1,109 @@
+package com.serotonin.mango.util;
+
+import br.org.scadabr.vo.scripting.ScriptVO;
+import com.serotonin.mango.rt.dataImage.DataPointRT;
+import com.serotonin.mango.rt.dataSource.DataSourceRT;
+import com.serotonin.mango.vo.DataPointVO;
+import com.serotonin.mango.vo.link.PointLinkVO;
+
+import java.text.MessageFormat;
+
+public final class LoggingScriptUtils {
+
+    private LoggingScriptUtils() {}
+
+    public static String infoErrorExecutionScript(Exception e,
+                                                  DataPointRT dataPoint,
+                                                  DataSourceRT dataSource) {
+        String context = generateContext(dataPoint, dataSource);
+        return infoErrorExecutionScript(e, context);
+    }
+
+    public static String infoErrorExecutionScript(Exception e,
+                                                  PointLinkVO vo,
+                                                  DataPointRT target,
+                                                  DataPointRT source) {
+        String context = generateContext(vo, target, source);
+
+        return infoErrorExecutionScript(e, context);
+    }
+
+    public static String infoErrorExecutionScript(Exception e, ScriptVO<?> vo) {
+        String context = generateContext(vo);
+        return infoErrorExecutionScript(e, context);
+    }
+
+    public static String infoErrorExecutionScript(Exception e, String context) {
+        if(e != null) {
+            String info = "Problem with execute script: context: {0}, {1}";
+            return MessageFormat.format(info, context, exceptionInfo(e));
+        } else {
+            String info = "Problem with execute script: context: {0}";
```

Comment:
I propose changing message to "Problem with executing script" or "Problem with script execution"